### ISSUE TITLE
Example appdata.xml is invalid, lacks required `<translation>` and `<summary>` tags

### ISSUE THREAD

**Issue by JonasT** [AppImage/AppImageKit#448]
The example appdata.xml generated by latest AppImage is invalid and won't validate with appstream-util even when filled out properly: it lacks required `<translation>` and `<summary>` tags.

**Comment by JonasT** [comment]
I tested some more and apparently, appimagetool uses something else than `appstream-util validate` and therefore by itself has no issue with `<translation>` and `<summary>` not being there.

However, maybe the default appimage.xml should validate with `appstream-util validate` too? I'm not sure, so I'll leave this ticket open for now

**Comment by probonopd** [comment]
We are currently using 

https://github.com/AppImage/AppImageKit/blob/cfa96e0b1c87fbd29e54fdcfae10ed2b88404359/appimagetool.c#L525-L526

**Comment by probonopd** [comment]
> The example appdata.xml generated by latest AppImage is invalid and won't validate with appstream-util even when filled out properly

cc @ximion

**Comment by ximion** [comment]
I can't comment on appstream-util, since that's not my tool (see #449), but a summary tag *is* mandatory, even when validating with `appstreamcli`. A "translation" tag however is not necessary and optional, so the validation shouldn't fail.

**Comment by JonasT** [comment]
Maybe the example appdata.xml generated should have a summary tag in such a case?

**Comment by ximion** [comment]
> Maybe the example appdata.xml generated should have a summary tag in such a case?

Yes, it should :-)

**Comment by azubieta** [comment]
Do we still generate example appdata.xml? If not please close this issue.

**Comment by TheAssassin** [comment]
AFAIK, we only validate using appstream-util, if that tool is available in `$PATH`.

**Comment by probonopd** [comment]
And `appstreamcli`:

https://github.com/AppImage/AppImageKit/blob/bf2652e3cbcb2574feb9ea298c0081aa467c1cfe/src/appimagetool.c#L586

That two such tools exist is a mess to begin with. The whole AppStream ecosystem suffers from inconsistent implementations, too many different versions, and overall complexity. Worst of all, the most recent version of the spec cannot even be verified using tools available for the oldest still-supported distributions. It's all very broken.

**Comment by JonasT** [comment]
Yes I agree, wasn't very fun to have two mismatched, incompatible tools. It would be better if there was a way to be guaranteed using a matched set of tools

**Comment by TheAssassin** [comment]
What do you mean, "two"?

**Comment by probonopd** [comment]
`appstreamcli` and `appstream-util`. Two tools to check for the "validity" of AppStream metainfo files, which happen to seldom agree...

**Comment by ximion** [comment]
That's because `appstreamcli` mostly answers the question "Is this thing valid AppStream and follows the spec?" while `appstream-util` in its default setting *also* has checks testing for additional style requirements set by GNOME [Software].
AppStream is fully backwards compatible, `appstreamcli` is as well, but of course older ascli versions don't know about additions to the spec and therefore will complain about them if they are encountered.

**Comment by TheAssassin** [comment]
Seems like we want to continue using `appstreamcli`.

**Comment by probonopd** [comment]
Looks like we are not that interested in checks testing for additional style requirements set by GNOME [Software] and should, hence, go with `appstreamcli` (only).
